packages/babel-plugin-lingui-macro/src/macroJsAst.ts

```diff
@@ -180,6 +180,8 @@ export function tokenizeChoiceComponent(
     format: format,
     options: {
       offset: undefined,
+      /** Default to fill other with empty value for compatibility with ICU spec */
+      other: "",
```

Comment:
oh, that means SWC plugins changes would also be required

Reply:
I see, I guess that's in a different repo 👀